### PR TITLE
Android SDK v3.5

### PR DESCRIPTION
**Pull request by dinuchiriac**
**Who:** me
**What:** Update WiFi Share demo app to the latest ChirpConnect SDK v3.5.0
**Why:** New SDK was recently released

### CODE REVIEW

File: ChirpWiFiShareAndroid/app/src/main/java/io/chirp/chirpwifi/MainActivity.java [hunk 1]
```diff
@@ -44,73 +45,120 @@ protected void onCreate(Bundle savedInstanceState) {
 
         status = findViewById(R.id.status);
         startStopListeningBtn = findViewById(R.id.startStopListening);
-
+        context = this;
+        startStopListeningBtn.setAlpha(.4f);
+        startStopListeningBtn.setClickable(false);
         /*
-        You can download licence string and credentials from your admin panel at admin.chirp.io
+        You can download config string and credentials from your admin panel at admin.chirp.io
```
Comment (joextodd) [inline ChirpWiFiShareAndroid/app/src/main/java/io/chirp/chirpwifi/MainActivity.java]: developers.chirp.io

**Review by joextodd — CHANGES_REQUESTED** [review]
just one minor change

File: ChirpWiFiShareAndroid/app/src/main/java/io/chirp/chirpwifi/MainActivity.java (hunk 1 above)
Comment (dinuchiriac) [reply ChirpWiFiShareAndroid/app/src/main/java/io/chirp/chirpwifi/MainActivity.java]: 👍

**Comment by dinuchiriac** [comment]
@joextodd   done

**Comment by dinuchiriac** [comment]
@joextodd  Included the `README` changes to reflect the latest updates, see the last commit.

**Review by joextodd — APPROVED** [review]

**Pull request merged**